modules/storage/redis-cache/README.md

```diff
@@ -82,4 +82,4 @@ module redisCache 'br/public:storage/redis-cache:2.0.1' = {
     ]
   }
 }
-```
\ No newline at end of file
+```
```

Comment:
Could you revert the change? It's a README for another module.

Reply:
I don't know why it is showing a change. I dont see any changes.

Reply:
run 'git checkout origin/main modules/storage/redis-cache/README.md' and try to commit the file back.

Reply:
done